api_tests/lib/ledgers/parity_instance.ts

```diff
@@ -1,27 +1,48 @@
 import { ChildProcess, spawn } from "child_process";
 import * as fs from "fs";
 import tmp from "tmp";
-import { LedgerInstance } from "./ledger_runner";
 import { LogReader } from "./log_reader";
 import { promisify } from "util";
 import { sleep } from "../utils";
+import getPort from "get-port";
+import { EthereumInstance } from "./ethereum";
+import { Logger } from "log4js";
 
 const openAsync = promisify(fs.open);
 
-export class ParityInstance implements LedgerInstance {
+export class ParityInstance implements EthereumInstance {
     private process: ChildProcess;
     private dbDir: any;
 
+    public static async new(
+        projectRoot: string,
+        logFile: string,
+        logger: Logger
+    ) {
+        return new ParityInstance(
+            projectRoot,
+            logFile,
+            logger,
+            await getPort({ port: 8545 }),
+            await getPort()
```

Comment:
```suggestion
            await getPort({ port: 30303 })
```

Reply:
I don't see the point in doing this. I also don't know why we are doing this everywhere else. I just want _some_ port, don't care which one it is.

Reply:
To me this port preference looked new and introduced by this PR hence my proposal. I did not realize it was not introduce by this PR.